Objects/longobject.c

```diff
@@ -3790,6 +3860,43 @@ l_divmod(PyLongObject *v, PyLongObject *w,
     return 0;
 }
 
+/* Compute
+ *     *pmod = v % w
+ * An error should be raised if NULL is passed to pmod.
+ * The caller owns a reference to pmod.
+ */
+static int
+l_mod(PyLongObject *v, PyLongObject *w, PyLongObject **pmod)
+{
+    PyLongObject *mod;
+
+    if (pmod == NULL) {
+        PyErr_SetString(PyExc_SystemError, "missing pmod argument to l_mod");
+        return -1;
+    }
```

Comment:
Sorry, I'm not following this. Isn't l_mod an internal function? If so, and `pmod != NULL` is a _precondition_ for calling it, then it suffices to just do

```
assert(pmod);
```

Reply:
Done.